app/modules/intelligence/provider/provider_service.py

```
@@ -421,10 +655,19 @@ def get_llm(self, agent_type: AgentProvider, config_type: str = "chat"):
         self._initialize_llm(config, agent_type)
         return self.llm
 
-    def get_pydantic_model(self) -> Model | None:
+    def get_pydantic_model(
```

Pylint (3.3.7)

[refactor] Too many return statements (7/6)

(R0911)